_posts/2020-08-10-datadog-backup.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: Backing up Datadog and Performing Bulk Edits
+tags:
+- datadog
+- monitoring
+- featured
+team: Core Infrastructure
+authors:
+- jimp
+- kamranf
```

Comment:
Some of our image rendering doesn't do so well with rendering multiple authors. 

I can fix  that, but `kamranf` needs an entry in `_data/authors.yml`